src/Symfony/Component/Yaml/Parser.php

```diff
@@ -359,6 +299,31 @@ public function parse($value, $flags = 0)
             mb_internal_encoding($mbEncoding);
         }
 
+        if (array_key_exists('<<', $data)) {
```

Comment:
It is not supported by the spec, it is built [on top of it](http://yaml.org/type/merge.html), so imo it should work like any other nodes and multiple occurrences should not be supported.